src/base/action.lua

```diff
@@ -68,6 +70,20 @@
 		end
 
 		action._list[act.trigger] = act
+
+		-- Add aliases table
+		if act.aliases then
+			table.foreachi(act.aliases, function(alias)
+				action._aliases[alias] = act.trigger
+			end)
+		end
+
+		-- Add deprecated aliases table
+		if act.deprecatedaliases then
+			for key, value in pairs(act.deprecatedaliases) do
+				action._deprecatedaliases[key] = value
+			end
```

Comment:
I think this needs to do one of two things; either automatically register deprecated aliases so they can be specified without the `aliases` table, or, emit a warning/error when the deprecated alias isn't in the aliases table.

My vote is for the auto registration because it's easier to work with.

Reply:
Sounds good to me

Reply:
I think I'm going to go with both a notify and patching it into the table.